app/src/main/res/drawable/cursor_color.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <size android:width="@dimen/cursor_width" />
+    <solid android:color="@color/format_text" />
```

Comment:
why do you use color created for a text to set it to the cursor? create your own cursor color